ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ImageLoader.qml

```diff
@@ -9,9 +9,10 @@ Item {
     property bool isCurrentUser: false
     property url source
     property bool playing: true
-    property bool isAnimated: true
+    property bool isAnimated: !!source && source.toString().endsWith('.gif')
```

Comment:
@emizzle I know you were the one who changed this to be `true` by default, but it actually didn't work. I think maybe you were missing a commit, because `isAnimated` never changed.

Let me know if this is a problem with the Gif unfurling you added. 
I tried to test Tenor and Giphy gifs, but they seems broken on master so I couldn't